cli/upload/upload.go

```diff
@@ -0,0 +1,149 @@
+package upload
+
+import (
+	"context"
+	"errors"
+	"flag"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/buildbuddy-io/buildbuddy/cli/arg"
+	"github.com/buildbuddy-io/buildbuddy/cli/log"
+	"github.com/buildbuddy-io/buildbuddy/cli/storage"
+	"github.com/buildbuddy-io/buildbuddy/server/remote_cache/cachetools"
+	"github.com/buildbuddy-io/buildbuddy/server/remote_cache/digest"
+	"github.com/buildbuddy-io/buildbuddy/server/util/grpc_client"
+	"google.golang.org/grpc/metadata"
+
+	repb "github.com/buildbuddy-io/buildbuddy/proto/remote_execution"
+	rspb "github.com/buildbuddy-io/buildbuddy/proto/resource"
+	bspb "google.golang.org/genproto/googleapis/bytestream"
+)
+
+var (
+	flags = flag.NewFlagSet("upload", flag.ContinueOnError)
+
+	target             = flags.String("target", "grpcs://remote.buildbuddy.io", "Cache gRPC target")
```

Comment:
nit: this naming is slightly confusing with `bazel build targets` (such as `//my/target:a`)

Would something such as `remote_instance_url` or `remote_instance_host` work better?

Reply:
This matches [download.](https://cs.github.com/buildbuddy-io/buildbuddy/blob/f1f111595aef724bf871f9cf8c8526992153aaf4/cli/download/download.go?q=cli%2Fdownload%2Fdownload.go#L29)

I think most people won't change this so it's probably not a big deal, but if we did want to change it I'd prefer something short and possibly matching bazel, like "remote_cache".